docs/lago-tutorial/templates-repo/office.json

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
As we said, these files are not needed anymore, can be replaced by passing one of:

   http://templates.ovirt.org/repo/repo.metadata (or acutally not passing any, this is the default)
   http://10.35.18.63/repo/repo.metadata (for the office-specific one)